### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by patrick-holden**

### CODE REVIEW

File: CSS/styles.css [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+  top: 60%;
+  bottom: 0;
+  right: 0;
+  opacity: 0;
+  -webkit-transition: all 0.4s ease-in-out 0s;
+  -moz-transition: all 0.4s ease-in-out 0s;
+  transition: all 0.4s ease-in-out 0s;
+}
+
+.content:hover .content-overlay{
```
Comment (gracesessions) [inline CSS/styles.css:184]: space before bracket

File: CSS/styles.css [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+  opacity: 0.8;
+}
+
+.content-image{
```
Comment (gracesessions) [inline CSS/styles.css:188]: space before bracket

File: CSS/styles.css [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+  opacity: 0;
+  -webkit-transform: translate(-50%, -50%);
+  -moz-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  -webkit-transition: all 0.3s ease-in-out 0s;
+  -moz-transition: all 0.3s ease-in-out 0s;
+  transition: all 0.3s ease-in-out 0s;
+}
+
+.content:hover .content-details{
```
Comment (gracesessions) [inline CSS/styles.css:209]: space before bracket

File: CSS/styles.css [hunk 4]
```diff
@@ -0,0 +1,472 @@
...
+  top: 50%;
+  left: 50%;
+  opacity: 1;
+}
+
+.content-details h3{
```
Comment (gracesessions) [inline CSS/styles.css:215]: space

File: CSS/styles.css [hunk 5]
```diff
@@ -0,0 +1,472 @@
...
+  font-weight: 500;
+  letter-spacing: 0.1rem;
+  margin-bottom: 0rem;
+  padding-top: 7.5rem;
+  padding-bottom: 0rem;
+  text-align: left;
+  text-transform: uppercase;
+}
+
+.content-details p{
```
Comment (gracesessions) [inline CSS/styles.css:226]: space

File: CSS/styles.css [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+  color: #ffffff;
+  font-size: 1rem;
+  text-align: left;
+  margin-top: 0.1rem;
+}
+
+.fadeIn-bottom{
```
Comment (gracesessions) [inline CSS/styles.css:233]: space

File: CSS/styles.css [hunk 7]
```diff
@@ -0,0 +1,472 @@
...
+from {
+  -webkit-transform: translateX(0);
+  transform: translateX(0);
+  }
+to {
+  -webkit-transform: translateX(-100%);
+          transform: translateX(-100%);
+  }
+}
+
```
Comment (gracesessions) [inline CSS/styles.css:397]: can delete line?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="CSS/normalize.css" rel="stylesheet">	
+	<link href="CSS/styles.css" rel="stylesheet">
+  <link rel="icon" type="image/x-icon" href="./Images/Git-Hub-ident-trans.png">
```
Comment (gracesessions) [inline index.html:10]: this line looks like it's indented different to others - change to all spaces?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+	<title>Page Title</title>
+</head>
+
+<body>
+	<header>
+    <nav>
+      <ul>
+        <li><a href="http://www.github.com/patrick-holden"><img src="./Images/Git-Hub-ident-trans.png" id="logo" alt="A link to my GitHub account"></a></li>
+        <li><a href="#portfolio-panel">Portfolio</a></li>
+				<li><a href="#about-contact">Contact Info</a></li>
```
Comment (gracesessions) [inline index.html:20]: same here with inconsistent indents

File: index.html [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+        <h2>Patrick<br>Designer <span class="pink">*</span> Developer</h2>
+      </div>
+      <div class="arrow bounce"></div>
+      <div class="intro">
+        <p>I'm a trainee web developer currently at iO Academy.<br>  Here I'll be posting about some of the things I've been building recently.</p>
+      </div>  
+    </section>
+    <section id="portfolio-panel" class="scroll-snap-always">
+      <h2>
+        Recent Work
```
Comment (gracesessions) [inline index.html:40]: could put all on same line within h2 tag

File: index.html [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+              <p class="content-text">My latest project</p>
+            </div>
+          </a>
+        </div>
+      </div>  
+    </section>
+
+    <section class="updates">
+      <h2>
+        Updates
```
Comment (gracesessions) [inline index.html:108]: could put all on same line

File: index.html [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+      </div>
+    </section>
+
+    <section id="about-contact">
+      <h2>
+        Contact
+      </h2>
+      <div id="about-contact-grid">
+        <p class="about-grid">
+          Hope you enjoyed looking through the site.  I update it regularly with new projects and builds. <br><br> Please feel free to drop me a line, if you'd like to get in touch.
```
Comment (gracesessions) [inline index.html:121]: same line?

File: index.html (hunk 10 above)
Comment (patrick-holden) [reply index.html:40]: On this occasion, I'll leave it as it is.

File: CSS/styles.css [hunk 13]
```diff
@@ -0,0 +1,472 @@
...
+  grid-row-end: 6;
+}
+
+.email-grid {
+  grid-column-start: 6;
+  grid-column-end: 7;
+  grid-row-start: 5;
+  grid-row-end: 6;
+}
+
```
Comment (gracesessions) [inline CSS/styles.css:342]: could delete line

**Pull request merged**